models/event.js

```diff
@@ -60,7 +60,16 @@ const MODEL = {
         .example({
             nodes: [{ name: '~commit' }, { name: 'main' }, { name: 'publish' }],
             edges: [{ src: '~commit', dest: 'main' }, { src: 'main', dest: 'publish' }]
+        }),
+    pr: Joi
```

Comment:
Since you're putting the same Joi schema in the `scm` already, you can just put `pr: Scm.pr` here (like https://github.com/screwdriver-cd/data-schema/blob/master/models/event.js#L25) instead of duplicating the schema.